client/tinychain/ml/mlr.py

```diff
@@ -0,0 +1,95 @@
+
+
+from tinychain.collection.tensor import einsum, Dense , Tensor
+from tinychain.ml import Layer, NeuralNet
+from tinychain.ref import After
+import tinychain as tc 
+import numpy as np
+
+
+class MultipleLinearRegression :
+    
+    @classmethod
+    def load (self,learningrate): 
```

Comment:
A `classmethod` doesn't have a `self` parameter, also I think this `load` method is basically redundant with `MultipleLinearRegression.__init__`